README.md

```diff
@@ -40,12 +40,20 @@ An example for a "porcelain" API would be [ROS2](https://www.ros.org/). Others a
 | [RTA-VRTE](https://www.etas.com/en/products/rta-vrte.php) | [Adaptive AUTOSAR](https://www.autosar.org/standards/adaptive-platform/) platform software framework for vehicle computer from [ETAS GmbH](https://www.etas.com) |
 | [Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) | Performant and robust open-source DDS implementation maintained by [ADLINK Technology Inc.](https://www.adlinktech.com/) |
 
-### Supported Platforms
+### Targeted quality levels & platforms
 
- * Linux
- * QNX
- * macOS (initial support - please test and report bugs)
- * Windows 10 (not yet - currently in progress)
+> [Quality level](./CONTRIBUTING.md#quality-levels) are 5..1, where 1 is highest
+
+|CMake project/target   | QNX  | Linux, Windows, MacOS | Comment                             |
+|-----------------------|:----:|:---------------------:|:-----------------------------------:|
+| example_icedelivery   | 4    | 4                     |                                     |
+| example_iceperf       | 4    | 4                     |                                     |
+| example_singleprocess | 4    | 4                     |                                     |
+| iceoryx_dds           | 4    | 4                     |                                     |
+| iceoryx_meta          | 4    | 4                     |                                     |
+| iceoryx_posh          | 1, 2 | 4                     | Will be split into separate targets |
+| iceoryx_utils         | 1    | 4                     |                                     |
+| iceoryx_introspection | 4    | 4                     |                                     |
```

Comment:
shouldn't this be `iox-introspecton-client`? It might confuse people since we also have introspection parts in iceoryx_posh

Reply:
I would use the names of the CMake targets/projects `project(iceoryx_introspection ..)`. I guess the introspection parts of RouDi should be move to the other CMake target. What do you think?

Reply:
yes, ideally there should be only one target per CMakeLists.txt and especially in iceoryx_posh it's confusing what's all included there